frontend/src/metabase/visualizations/echarts/cartesian/model/transforms.ts

```diff
@@ -0,0 +1,45 @@
+import type { NumericScale, StackType } from "metabase-types/api";
+import { isNumber } from "metabase/lib/types";
+import type { NumericAxisScaleTransforms } from "./types";
+
+function getSign(value: number) {
+  return value >= 0 ? 1 : -1;
+}
+
+function signedSquareRoot(value: number) {
+  return getSign(value) * Math.sqrt(Math.abs(value));
+}
+
+export function getYAxisScaleTransforms(
```

Comment:
We can put `getAxisTransforms` from `axis.ts` here and use it unchanged I believe to pass the transforms to the data transformation

Reply:
Sure we can use the same function, although it will still need the stack type check, see my comment below.